src/test/java/ezschedule/model/event/EventTest.java

```diff
@@ -10,93 +14,132 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-import org.junit.jupiter.api.Assertions;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 import org.junit.jupiter.api.Test;
 
 import ezschedule.testutil.EventBuilder;
 
 public class EventTest {
 
     @Test
-    public void isSameEvent() {
-        // same object -> returns true
-        assertEquals(EVENT_A, EVENT_A);
+    public void isCompleted() {
+        LocalDate today = LocalDate.now();
```

Comment:
Perhaps for the sake of testing, `today` is fixed to a certain date?
(Not too sure how feasible this is)

Reply:
@lhy-hoyin today cannot be fixed to a certain date as methods such as isPastDate() and isFutureDate() within isCompleted uses LocalDate.now() as a comparison. Fixing it to a specific date will lead to the else case within isCompleted to be unreachable.